cuda/private/rules/cuda_library.bzl

```diff
@@ -90,8 +92,8 @@ def _cuda_library_impl(ctx):
             pic_objects = pic_objects,
         ),
         CcInfo(
-            compilation_context = compilation_ctx,
-            linking_context = linking_ctx,
+            compilation_context = cc_info.compilation_context,
+            linking_context = cc_info.linking_context,
         ),
```

Comment:
IIRC simply returning cc_info here didn't seem to work but not sure why?

Reply:
Maybe a bug in bazel itself, let ignore it ;)